src/pages/agent/tests-to-run/tests-to-run.tsx

```diff
@@ -39,6 +40,79 @@ interface Props {
   agentType?: string;
 }
 
+const columns = [
+  {
+    Header: "Name",
+    accessor: "overview.details.name",
+    textAlign: "left",
+    filterable: true,
+  },
+  {
+    Header: "Path",
+    accessor: "overview.details.path",
+    textAlign: "left",
+    filterable: true,
+  },
+  {
+    Header: "Test type",
+    accessor: "type",
+    Cell: ({ value }: any) => (
+      <>
+        {capitalize(value)}
+      </>
+    ),
+    textAlign: "left",
+  },
+  {
+    Header: "State",
+    accessor: "overview.result",
+    Cell: ({ row: { original: { toRun } } }: any) => (
+      <span tw="leading-64">
+        {toRun
+          ? "To run"
+          : <span tw="font-bold text-green-default">Done</span>}
+      </span>
+    ),
+    textAlign: "left",
+  },
+  {
+    Header: "Coverage, %",
+    accessor: "coverage.percentage",
+    Cell: ({ value, row: { original: { toRun } } }: any) => (toRun ? null : <Cells.Coverage tw="inline" value={value} />),
+  },
+  {
+    Header: "Methods covered",
+    accessor: "coverage.methodCount.covered",
+    Cell: ({
+      value,
+      row: { original: { id = "", toRun = false, coverage: { methodCount: { covered = 0 } = {} } = {} } = {} },
+    }: any) => (
+      toRun ? null : (
+        <Cells.Clickable
+          tw="inline"
+          disabled={!value}
+        >
+          <Link to={getModalPath({ name: "coveredMethods", params: { coveredMethods: covered, testId: id } })}>
+            {value}
+          </Link>
+        </Cells.Clickable>
+      )
+    ),
+  },
+  {
+    Header: "Duration",
+    accessor: "overview.duration",
+    Cell: ({ value, row: { original: { toRun } } }: any) => (toRun ? null : <Cells.Duration value={value} />),
+  }];
+
+const stub = (
```

Comment:
No, we need a return ternary operator for the stubs. But we need to remove the ternary operator inside of the second stub